flink-libraries/flink-table/src/test/scala/org/apache/flink/table/functions/aggfunctions/AggFunctionTestBase.scala

```diff
@@ -56,11 +53,15 @@ abstract class AggFunctionTestBase[T] {
 
     if (ifMethodExitInFunction("merge", aggregator)) {
       // iterate over input sets
-      for((vals, expected) <- inputValueSets.zip(expectedResults)) {
+      for ((vals, expected) <- inputValueSets.zip(expectedResults)) {
         val (firstVals, secondVals) = vals.splitAt(vals.length / 2)
```

Comment:
In addition to splitting the `vals` data set, we should also test correctness, if we merge with an empty accumulator, i.e., aggregate all `vals` into one accumulator and merge it with an empty one.